ui/src/components/Configuration/ConfigurationAdvancedPanel.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <FeatherExpansionPanel
+    id="advanced-panel"
+    class="expansion-panel advanced-panel"
+    title="Advanced Options (optional)"
+    :modelValue="props.active"
+    @update:modelValue="props.activeUpdate"
+  >
+    <div>
+      <div
+        v-bind:key="index"
+        v-for="(item, index) in props.items"
+        class="item-wrapper"
+      >
+        <FeatherAutocomplete
+          type="single"
+          label="Key"
+          textProp="name"
+          @search="(query: string) => search(query, props.type, props.subType, index)"
+          v-model="item.key"
+          @update:modelValue="(key: { hint: string }, index: number): any => {
+            ConfigurationHelper.forceSetHint(key, index);
+            props.advancedKeyUpdate(key, index)
+          }"
+          :results="results.list[index]"
+        ></FeatherAutocomplete>
+        <!-- Blank space ' ' below is part of forceSetHint() workaround for FeatherInput.
+            If item.hint is blank on initial load, it will not render the internal element we need
+            for forced update. So when item.hint is empty, we supply an empty space which is enough
+            to force FeatherInput to render the help label.
+        -->
+        <FeatherInput
+          class="hint-label"
+          label="Value"
+          :hint="item.hint || ' '"
+          v-model="item.value"
+        />
+        <FeatherButton
+          icon="Delete"
+          @click="() => deleteAdvancedOption(index)"
+        >
+          <FeatherIcon
+            class="delete-icon"
+            :icon="Delete"
+          ></FeatherIcon>
+        </FeatherButton>
+      </div>
+      <div class="button-wrapper">
+        <FeatherButton
+          :disabled="buttonAddDisabled"
+          @click="addAdvancedOption"
+          primary
+          >Add</FeatherButton
+        >
+      </div>
+    </div>
+  </FeatherExpansionPanel>
+</template>
+
+<script setup lang="ts">
+import { reactive, PropType, watch, computed } from 'vue'
```

Comment:
Now that we have auto imports we can remove any imports from vue/vue-router/vueuse unless they are types.
So this could change to `import { PropType } from 'vue'` here. Elsewhere in this PR the vue imports can be removed entirely.

Reply:
`DefineComponent` as well right?

Reply:
Yup, any `type` will still need to be imported